sklearn/metrics/tests/test_pairwise.py

```diff
@@ -245,13 +254,26 @@ def callable_rbf_kernel(x, y, **kwds):
 @pytest.mark.parametrize(
         'func, metric, kwds',
         [(pairwise_distances, 'euclidean', {}),
+         (pairwise_distances, minkowski, _minkowski_kwds),
```

Comment:
We can include the skip here:

```suggestion
         pytest.param(pairwise_distances, minkowski, _minkowski_kwds,
         			  marks=pytest.mark.skipif(sp_version < parse_version("1.0")),
         			  reason="minkoski does not accept ..."),
```

Reply:
I guess this kind of parametrization should be applied to the 'minkowsi' case too, right?